Lib/test/test_xmlrpc.py

```diff
@@ -343,77 +343,71 @@ def run_server():
             self.assertEqual(p.method(), 5)
             self.assertEqual(p.method(), 5)
 
-    def test_simple_xml_rpc_dispatcher_resolves_registered_func_and_does_not_chain_excs_when_calling_it(self):
-        exp_params = 1, 2, 3
-        dispatched = False
 
-        class DispatchExc(Exception):
-            """Raised inside the dispatched func"""
+class SimpleXMLRPCDispatcherTestCase(unittest.TestCase):
+    class DispatchExc(Exception):
+        """Raised inside the dispatched functions when checking for chained exceptions"""
+
+        def __init__(self, *args, method=None, params=None, **kwargs):
```

Comment:
I really do prefer keyword arguments and keyword structure, but hey, I learned something new about `Exception.args` and these are just tests and the code that instantiates the exception and the code that processes it are very close, so it is ok